tests/third_party/cupy/indexing_tests/test_insert.py

```diff
@@ -83,12 +83,12 @@ class TestPut(unittest.TestCase):
     def test_put(self, xp, dtype):
         a = testing.shaped_arange(self.shape, xp, dtype)
         # Take care so that actual indices don't overlap.
-        if self.mode == "raise":
-            inds = xp.array([2, -1, 3, 0])
+        if self.mode in ("raise", "wrap"):
```

Comment:
raised gh-1365 in dpctl to clarify the behavior with `mode=wrap`

Reply:
it would be great to add some comment with a link to github issue in the comment